client/me/help/help-contact-closed/index.jsx

```diff
@@ -7,45 +7,44 @@
 import React from 'react';
 import i18n, { localize } from 'i18n-calypso';
 
-const christmas2017ClosureStartsAt = i18n.moment( 'Sun, 24 Dec 2017 00:00:00 +0000' );
-const newYear2018NoticeStartsAt = i18n.moment( 'Fri, 29 Dec 2017 00:00:00 +0000' );
-const newYear2018ClosureStartsAt = i18n.moment( 'Sun, 31 Dec 2017 00:00:00 +0000' );
-
 /**
  * Internal dependencies
  */
+import FoldableCard from 'components/foldable-card';
 import FormSectionHeading from 'components/forms/form-section-heading';
 
-const HelpContactClosed = ( { translate } ) => {
+const easter2018ClosureStartsAt = i18n.moment( 'Sun, 1 Apr 2018 00:00:00 +0000' );
+
+const HelpContactClosed = ( { compact, translate } ) => {
 	const currentDate = i18n.moment();
 	let closureHeading;
 	let closureMessage;
 
-	if ( currentDate >= newYear2018ClosureStartsAt ) {
-		closureHeading = translate( "Limited Support New Year's Day" );
+	if ( currentDate <= easter2018ClosureStartsAt ) {
+		closureHeading = translate( 'Limited Support over Easter' );
 		closureMessage = translate(
-			"Live chat is closed today for the New Year's Day holiday. If you need to get in touch with us, submit " +
-				"a support request below and we'll respond by email. Live chat will reopen on January 2nd. Thank you!"
-		);
-	} else if ( currentDate >= newYear2018NoticeStartsAt ) {
-		closureHeading = translate( "Limited Support New Year's Day" );
-		closureMessage = translate(
-			"Live chat will be closed on Monday, January 1, 2018 for the New Year's Day holiday. If you need " +
-				"to get in touch with us, you’ll be able to submit a support request from this page and we'll " +
-				'respond by email. Live chat will reopen on January 2nd. Thank you!'
-		);
-	} else if ( currentDate >= christmas2017ClosureStartsAt ) {
-		closureHeading = translate( 'Limited Support over Christmas' );
-		closureMessage = translate(
-			'Live chat is closed today for the Christmas holiday. If you need to get in touch with us, submit ' +
-				"a support request below and we'll respond by email. Live chat will reopen on December 26. Thank you!"
+			'Live chat will be closed on Sunday, April 1, 2018 for the Easter Sunday holiday. ' +
+				'If you need to get in touch with us, you’ll be able to submit a support request ' +
+				"from this page and we'll respond by email. Live chat will reopen on April 2nd. Thank you!"
 		);
 	} else {
-		closureHeading = translate( 'Limited Support over Christmas' );
+		closureHeading = translate( 'Limited Support over Easter' );
 		closureMessage = translate(
-			'Live chat will be closed on Sunday, December 24th and Monday, December 25th for the Christmas holiday. ' +
-				'If you need to get in touch with us, you’ll be able to submit a support request from this page and ' +
-				"we'll respond by email. Live chat will reopen on December 26th. Thank you!"
+			'Live chat is closed today for the Easter Sunday holiday. If you need to get in touch with us, submit a support ' +
```

Comment:
Could we find a way to keep these more generic? This requires re-translation and we're always one week late anyway. If there is no point in translating them, please don't wrap them in a `translate()` call. Thanks!

Reply:
Thanks for the suggestion @akirk — it's easy for me to add a `translate` without thinking about the practical implications of the time it takes for it to get translated.

/cc @sandymcfadden @Jstachurski @kevmarsden @lettergrade as folks who have requested Live Chat closure notices in the past 6 months — maybe we can look into some more standard closure notice that's always translated the same and just has some details swapped out?